Makefile

```diff
@@ -59,6 +59,13 @@ review:
 	$(eval export TF_VAR_static_route=$(shell script/get_next_mapping.sh ${PR_NAME}))
 	$(eval BACKEND_KEY=-backend-config=key=${PR_NAME}.tfstate)
 
+.PHONY: review_aks
+review_aks:
+	$(eval include global_config/review.sh)
+	$(if $(pr_id), , $(error Missing environment variable "pr_id"))
+	$(eval ENVIRONMENT=review-pr-$(pr_id))
+	$(eval export TF_VAR_environment=$(ENVIRONMENT))
+
```

Comment:
paas review apps run
`$(eval export TF_VAR_static_route=$(shell script/get_next_mapping.sh ${PR_NAME}))`
so we will have to investigate this requirement for aks